builder/docker/communicator.go

```diff
@@ -173,7 +173,7 @@ func (c *Communicator) UploadDir(dst string, src string, exclude []string) error
 
 	// Make the directory, then copy into it
 	cmd := &packer.RemoteCmd{
-		Command: fmt.Sprintf("set -e; mkdir -p %s; command cp -R %s/ %s",
+		Command: fmt.Sprintf("set -e; mkdir -p %s; command cp -R %s/. %s",
```

Comment:
No because ```mkdir -p``` isn't a valid command on Windows. This wasn't a request to make it work on Windows or anything. More just pointing it out just in case :smile: